Karl/test_service/http_subscriber.h

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
```

Comment:
This class is so awesome, perhaps move it into `Blocks/`, or even into `Sherlock/`? Can wait until Thailand too FWIW :)